### ISSUE TITLE
About Chinese translation

### ISSUE THREAD

**Issue by ZhouGeorge** [Hvass-Labs/TensorFlow-Tutorials#83]
The Chinese translation has not been updated for a long time.
I have updated the part of  tutorials after thrillerist‘s work and willing to update the rest part.
Could you recommend my link?(https://github.com/ZhouGeorge/TensorFlow-Tutorials)

Thank you for your contribution! Hope more people will benefit from your tutorial.

**Comment by Hvass-Labs** [comment]
Thanks very much, this is very cool!

Would it be possible to merge your code into @thrillerist github repo? I think he/she can give you write-permission to the repo so you can help maintain it in the future. You still get credit for your work by writing your name and GitHub link on top of every Notebook you worked on. I think it would be best to continue using the repo that everyone already knows instead of having two links.

Have you thought about doing Chinese versions of my YouTube tutorials? You just need to buy a decent microphone because sound quality is very important. I use Vokoscreen to record the videos on Linux and I use Resolve to edit them on Windows. I use a combination of free audio software to clean up the sound, but if you get a better microphone than mine, then you may not need that.

**Comment by ZhouGeorge** [comment]
Thank you for your advice. I will try to contact him.

I haven't considered making videos tutorials right now. I don't have enough time now, so I hope to finish the translation of  Notebooks first.

**Comment by Hvass-Labs** [comment]
@ZhouGeorge and @thrillerist

I have given this some thought. I think the best solution would be that I make a new repo under my account and give both of you collaborator rights on it, so you can push to the repo. Because that way I can add new collaborators in the future, if both of you no longer want to work on it. I think that is better than moving it from one repo to another all the time.

However, I cannot fork my own repo. Instead I can try and fork from this one: https://github.com/ZhouGeorge/TensorFlow-Tutorials and then rename it to TensorFlow-Tutorials-Chinese. And then give both of you collaborator rights on that.

You will not get the "like-stars" on your personal account, but you will get full credit in the top of every notebook with a link to your personal github page.

Do you agree with that idea? Or do you have a better solution?

**Comment by raineydavid** [comment]
@Hvass-Labs it might be worth creating a [github organisation account](https://help.github.com/articles/creating-a-new-organization-from-scratch/) then you can invite collaborators such as  @ZhouGeorge and @thrillerist and give permissions.

**Comment by ZhouGeorge** [comment]
@Hvass-Labs  I agree with it. This is a good solution for the long term.

**Comment by Hvass-Labs** [comment]
Actually, the best would be if @thrillerist could rename his fork to TensorFlow-Tutorials-Chinese and then transfer ownership to me, because then we would keep all the stars he received, and I can add new collaborators in the future. See https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/

However, this page https://help.github.com/articles/about-repository-transfers/ says that you can't transfer a repo to an account that already has a fork in the same network. But it's unclear if it works if the fork has been renamed.

@thrillerest could you try and rename your fork to TensorFlow-Tutorials-Chinese and transfer ownership to me?

@raineydavid Thanks for the tip. I thought about making an organization but I think it's overkill for this project. It should work fine by just adding collaborators to a normal github project.

**Comment by Hvass-Labs** [comment]
I have been unable to get in contact with @thrillerist so I hope he/she is alright.

I also contacted GitHub's support to ask what they recommend we might do, but I haven't heard from them.

@ZhouGeorge I was unable to fork your version of the TensorFlow-Tutorials repo because I already have such a repo in my account. I thought maybe I could fork it to a new name. Maybe you could have renamed it first, but I'm not sure that would work because the forks are in the same network. So instead I have made a new repo in my account named TensorFlow-Tutorials-Chinese and I have invited both you and thrillerist as collaborators, so you have write-access to that repo. I suggest you fork that repo and copy all the files over manually, then push it back. It's a bit messy and we loose the link between the original repo and the Chinese version. I thought about making a Chinese branch on the original repo, but that might make it a bit difficult for people to install and switch to that branch, and I'm also not happy about giving others write-access to the main repo. I also thought about simply having the Chinese versions in the main repo, but that would also be a bit messy, especially if the files become out of date.

So I think this is the best solution. Let me know if you have a better idea.

**Comment by raineydavid** [comment]
@ZhouGeorge @Hvass-Labs I have taken the initiative https://github.com/Tensorflow-Tutorials/ organisation.  I forked the original repo and I cloned and then pushed back up TensorFlow-Tutorials-CN  in that repo.  I have invited you both as members, if you want to pursue it. If not I shall continue curating other repos with Tensorflow (I am also a Data Scientist!) @Hvass-Labs you can add people as teams in organisations and you can also use a file called [CODEOWNERS](https://help.github.com/articles/about-codeowners/) which limits permissions even in [teams](https://help.github.com/articles/about-teams/). You can limit contribution to specific repos etc. Also github has a beta feature of showing your organisation contributions etc.

**Comment by Hvass-Labs** [comment]
@ZhouGeorge: TL;DR Please create the Chinese translation under https://github.com/Hvass-Labs/TensorFlow-Tutorials-Chinese as I suggested above.

@raineydavid I appreciate your enthusiasm and I don't want to crush your spirit, but I believe it would be best if I keep the entire project incl. translations under my personal account for now, and add collaborators as needed. Let me explain why:

1) These tutorials took a very long time to create. They are very polished and that's why people like them. A few times people have opened PR's where they added a whole bunch of stuff which messed up the tutorials. I of course politely refused to merge those PR's. The official TensorFlow tutorials from Google are also nowhere nearly as polished as mine. So apparently it's a special skill to create good tutorials and I don't trust random strangers to hold up to the high standard I've set here. That's why I want to keep 100% control of the original English version and I do not allow anyone else to have write access to that repo.
2) The original link https://github.com/Hvass-Labs/TensorFlow-Tutorials is used everywhere on the internet now, so it would be confusing if it were to change. (Perhaps GitHub can redirect, I don't know).
3) Making a translation is a very big job. Several people have told me they wanted to do it in e.g. Japanese and Russian. I told them to translate Tutorials 1, 2 and 3-C as a start and then make a repo so others could help with the rest. But it was so long ago and they haven't come back with even the first 3 tutorials translated. So I'm not holding my breath that there's going to be any more translations than this Chinese version, and we're very lucky that somebody has come along and translated the remaining ones.
4) The first Chinese translator has disappeared and we cannot reach him/her. That may very well happen to the current translator as well. And you may also disappear in a year or two, if you get bored with maintaining a library for which there is very little benefit for you. So it is important that I maintain full control of the repos / organization so I can assign others as needed.

The conclusion is that it currently seems like overkill to make a GitHub organization. So I would ask you to remove that again so it doesn't confuse people. If in the future there is a need for it, then I will create an organization that I control but I may ask you to help organize it, if you're still interested at that time.

But thanks again for your enthusiasm! :-)

**Comment by raineydavid** [comment]
@Hvass-Labs The [Tensorflow Tutorials organisation](https://github.com/Tensorflow-Tutorials) is now curating tensorflow tutorials across github/ open source and makes it clear @ZhouGeorge version in the organisation clone with reference to his original repository and your Tensorflow-Tutorial repo on the org is clearly a fork.  Hopefully it could inspire me and others to either write our own or learn from different ways of doing the same thing.  Good luck with the project and thanks for writing an awesome tutorial.

**Comment by Hvass-Labs** [comment]
@ZhouGeorge Where did you decide to upload your translated version? I can't see it here: https://github.com/Hvass-Labs/TensorFlow-Tutorials-Chinese

Please note that I have nothing to do with this organization: https://github.com/Tensorflow-Tutorials and I will NOT move my own repo there and I am not an organizer there, so I cannot add new collaborators to update the Chinese translations in the future. I still think it would be better if you place your translated version here: https://github.com/Hvass-Labs/TensorFlow-Tutorials-Chinese

@raineydavid I think a list of ordinary links would have been better. Now people might get confused and think those repos are the real ones and use outdated code when you get tired of maintaining it.
